crates/rolldown/src/utils/ecma_visitors/mod.rs

```diff
@@ -0,0 +1,48 @@
+use oxc::{
+  ast::{visit::walk_mut, VisitMut},
+  span::{GetSpanMut, Span},
+};
+use rustc_hash::FxHashSet;
+
+/// Make sure there aren't any duplicate spans in the AST.
+pub struct EnsureSpanUniqueness {
```

Comment:
Maybe rename to `EnsureModuleDeclarationSpanUniqueness`?

Reply:
It's not just for `ModuleDeclaration`s. Other nodes type would be added after https://github.com/oxc-project/oxc/issues/4799.

Reply:
I see